### PR TITLE
feat: add chat.fakerjs.dev subdomain redirection to discord

### LINKED ISSUE #295: chat.fakerjs.dev
### Clear and concise description of the problem

Several projects redirect chat.project.dev to discord, and it is a useful convention.

It would also be nice to have social links in the landing of fakerjs.dev, and a Community section in the Getting Started guide

### Suggested solution

Support https://chat.fakerjs.dev/

### Alternative

_No response_

### Additional context

Y'all are awesome, thanks for the work you are doing with faker-js!

### PR DESCRIPTION
**Pull request by ejcheng**
Fixes #295

### CODE REVIEW

**Review by Shinigami92 — CHANGES_REQUESTED** [review]
You need to replace all links like in markdown files with new url

**Comment by import-brain** [comment]
> You need to replace all links like in markdown files with new url

Could you please clarify what you mean by this? I interpreted your instructions as "replace all instances of fakerjs.dev with chat.fakerjs.dev in markdown files", but I don't think that makes sense.

**Comment by Shinigami92** [comment]
https://github.com/faker-js/faker/blob/3242dfc4b2f566931c3687173f6361a6384b6247/README.md?plain=1#L6

Just one example, please search over the whole repo

**Comment by import-brain** [comment]
> [`3242dfc`/README.md?plain=1#L6](https://github.com/faker-js/faker/blob/3242dfc4b2f566931c3687173f6361a6384b6247/README.md?plain=1#L6)
> 
> Just one example, please search over the whole repo

Ohhh I see, will do.

**Comment by import-brain** [comment]
Whoops, misclick.

File: netlify.toml [hunk 1]
```diff
@@ -4,3 +4,8 @@
 [build]
   publish = "docs/.vitepress/dist"
   command = "npx pnpm i --store=node_modules/.pnpm-store && npm run docs:build:ci"
+[[redirects]]
+from = "https://chat.fakerjs.dev"
+to = "https://discord.com/invite/4qDjAmDj4P"
+status = 301
+force = true
```
Comment (ST-DDT) [inline netlify.toml:11]: Is the formatting here correct?
The section above looks slightly different.

Comment (import-brain) [reply netlify.toml:11]: @ST-DDT This is what I used to write this formatting, it looks a bit different than the sections above.

https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file

Comment (ST-DDT) [reply netlify.toml:11]: I will approve this once it uses consistent formatting. The example uses indented formatting, so we should go for that
```suggestion
[[redirects]]
  from = "https://chat.fakerjs.dev"
  to = "https://discord.com/invite/4qDjAmDj4P"
  status = 301
  force = true
```

Comment (import-brain) [reply netlify.toml:11]: > I will approve this once it uses consistent formatting. The example uses indented formatting, so we should go for that
> 
> ```suggestion
> 
> [[redirects]]
> 
>   from = "https://chat.fakerjs.dev"
> 
>   to = "https://discord.com/invite/4qDjAmDj4P"
> 
>   status = 301
> 
>   force = true
> 
> ```

Oh, I thought you were referencing the double brackets around "redirects". Committed

**Review by ST-DDT — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**